src/app/game/game-form/game-form.component.html

```diff
@@ -0,0 +1,18 @@
+<input 
+    class="border border-gray-300 text-sm rounded-md focus:border-blue-500 block w-full p-2.5" 
+    type="text" 
+    placeholder="Nome" 
+    minlength="3"
```

Comment:
Usar validação no arquivo .ts
https://angular.io/guide/form-validation#validating-input-in-reactive-forms